### ISSUE TITLE
Another date problem 😅

### ISSUE THREAD

**Issue by bitboxer** [dkarter/cookie_monster#15]
So my little side project (wishlephant.com) might be a magnet for weird cookie date formats 😅.

Found another one.

```
__cflb=__REDACTED__; SameSite=Lax; path=/; expires=Fri, 20-Oct-23 07:22:39 GMT; HttpOnly
```

That's coming from this URL: https://www.trendyol.com/de/trendyol-collection/pullover-schwarz-normal-p-247657167?v=s

I can add the test and fix the parser for that on the weekend if you like.

**Comment by dkarter** [comment]
Fixed in v1.1.5, thanks for the PR!

For now I think this is ok - especially if browsers support this format, but long term it might make more sense to expose configuration options for `additional_datetime_parsers` for handling non standard-compliant datetime parsers. Or alternatively, add an option for `decode` to not parse the expires.

**Comment by bitboxer** [comment]
Yes, I think now it has all the basic permutations of date formats in it. 🤞 that I don't discover anything else out there. If I do, I will create an issue and then we discuss how this should be fixed. A "do not raise errors if you cannot decode the date" would be a nice fallback.
